### PR TITLE
assignment 7 warmup

### PR DESCRIPTION
**Pull request by gpsingh12**
Prof, 
Please provide feedback on task_05. If I am using .append it gives me the elements in reverse order internally. Not getting right.
Thanks

### CODE REVIEW

File: task_05.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+"""Module uses for loop and proporties
+   of lists to reverse elements of list.
+"""
+def flip_keys(to_flip):
+    """
+    """
+    #to_flip[::-1]
+    counter = []
+    for item in to_flip:
+       counter.extend(item[::-1])
```
Comment (cheuschober) [inline task_05.py]: @gpsingh12, So, first things first, you should be incrementing your counter (an integer) as it's suggested in the homework hint. Then print it. It should just be an integer to start, 0 in this case. Notice how, as it increments, it resembles the index of your list. Now think back to task 2. how did you *replace* a value of an item in a list? Not appending or extending -- replacement? You need the index to do that so use your counter as one!

Comment (gpsingh12) [inline task_05.py]: The error now is [(1, 2, 3)] is not [(1, 2, 3)

Comment (cheuschober) [inline task_05.py]: @gpsingh12, have you pushed these commits? I can't see what you've done to offer any additional advice.

Comment (gpsingh12) [inline task_05.py]: Prof.
I just pushed the changes to github.

**Pull request merged**